sections/docker/multi_stage_builds.md

```diff
@@ -0,0 +1,75 @@
+# Use multi-stage builds
+
+### One Paragraph Explainer
+
+Multi-stage builds allow to separate build- and runtime-specific environment details, such as available binaries, exposed environment variables, and even the underlying operating system.
+Splitting up your Dockerfiles into multiple stages will help to reduce final image and container size as you'll only ship what you really need to run your application. Sometimes, you'll need to
+include tools that are only needed during the build phase, for example development dependencies such as the TypeScript CLI. You can install it during the build stage and only use the final output in the run stage.
```

Comment:
Same thing here, it seems there are linebreaks in the middle of sentences in this block